Subject: powerpc: Make bootwrapper verbose when KBUILD_VERBOSE.

Add '-x' to the envocation of the bootwrapper build script when
KBUILD_VERBOSE is not zero.

???Will this only work for bash???

Signed-off-by: Geoff Levand <geoffrey.levand@am.sony.com>
---
 arch/powerpc/boot/Makefile |    9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

--- a/arch/powerpc/boot/Makefile	2007-08-27 14:01:20.000000000 -0400
+++ b/arch/powerpc/boot/Makefile	2007-08-27 14:01:21.000000000 -0400
@@ -120,10 +120,15 @@ CROSSWRAP := -C "$(CROSS_COMPILE)"
 endif
 endif
 
+ifneq ($(KBUILD_VERBOSE),0)
+verbose_shell := -x
+endif
+
 # args (to if_changed): 1 = (this rule), 2 = platform, 3 = dts 4=dtb 5=initrd
 quiet_cmd_wrap	= WRAP    $@
-      cmd_wrap	=$(CONFIG_SHELL) $(wrapper) -c -o $@ -p $2 $(CROSSWRAP) \
-		$(if $3, -s $3)$(if $4, -d $4)$(if $5, -i $5) vmlinux
+      cmd_wrap	=$(CONFIG_SHELL) $(verbose_shell) $(wrapper) -c -o $@ -p $2 \
+		$(CROSSWRAP) $(if $3, -s $3)$(if $4, -d $4)$(if $5, -i $5) \
+		vmlinux
 
 image-$(CONFIG_PPC_PSERIES)		+= zImage.pseries
 image-$(CONFIG_PPC_MAPLE)		+= zImage.pseries
